src/main.rs

```diff
@@ -20,8 +20,19 @@
 #[macro_use]
 extern crate slog;
 
+use clap;
+use lazy_static;
+use native_tls;
+use slog_async;
+use slog_term;
+use tasked_futures;
+use tokio_core;
+use tokio_tls;
+use url;
+
 use clap::{App, Arg};
 
+use futures;
```

Comment:
All these new imports are redundant. Why did you add them?

Reply:
These are issues that were solved in a previous PR, I have not rebased correctly so you are seeing ~5 (?) commits that have previously been pulled. Ill work on sorting this out.